From 77aa0aaff646a5838de56eba80c0de9df9ecad21 Mon Sep 17 00:00:00 2001
From: Patrick Rudolph <patrick.rudolph@9elements.com>
Date: Mon, 23 Oct 2023 14:09:08 +0200
Subject: [PATCH 06/13] configurations: sbp1: Add MAX6639

Add the air inlet temperature sensors that are connected to the
MAX6639 that also control the fan PWM.

Tested: All 24 temperature sensors are visible on the webui.

Change-Id: I4831a4cbf0f4e4b00b75e6d6d5bd58ab484e59eb
Signed-off-by: Patrick Rudolph <patrick.rudolph@9elements.com>
---
 configurations/sbp1_baseboard.json | 264 +++++++++++++++++++++++++++++
 1 file changed, 264 insertions(+)

diff --git a/configurations/sbp1_baseboard.json b/configurations/sbp1_baseboard.json
index cf8e351..aab4acc 100644
--- a/configurations/sbp1_baseboard.json
+++ b/configurations/sbp1_baseboard.json
@@ -2018,6 +2018,270 @@
             "in0_Name": "SSB_RSSD28 P12V Output Voltage",
             "in1_Name": "SSB_RSSD28 P3V3 Output Voltage"
         },
+        {
+            "Address": "0x2e",
+            "Bus": "105",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan01 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "118",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan02 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "106",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan03 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "119",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan04 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "107",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan05 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "120",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan06 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "108",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan07 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "121",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan08 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "109",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan09 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "123",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan10 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "110",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan11 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "124",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan12 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "111",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan13 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "125",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan14 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "112",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan15 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "126",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan16 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "114",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan17 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "127",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan18 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "115",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan19 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "128",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan20 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "116",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan21 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "129",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan22 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "117",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan23 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
+        {
+            "Address": "0x2e",
+            "Bus": "130",
+            "Labels": [
+                "temp1"
+            ],
+            "Name": "Fan24 Temperature",
+            "PollRate": 5.0,
+            "PowerState": "Always",
+            "Type": "MAX6639"
+        },
         {
             "Address": "0x58",
             "Bus": "98",
-- 
2.41.0

